.env.persistent-seeder

```diff
@@ -0,0 +1,39 @@
+## all
+NODE_ENV = 'production'
+
+## web3torrent, xstate-wallet, rps, simple-hub
+CHAIN_NETWORK_ID = '3'
+
+## rps, xstate-wallet, e2e-tests
+TARGET_NETWORK = 'ropsten'
+
+## e2e-tests
+HEADLESS=false
+USE_DAPPETEER=true
```

Comment:
Let's be consistent and add spaces around `=` and use string values?

Reply:
https://github.com/statechannels/monorepo/pull/1584/commits/203fad7437a9c48538cebbba379e59dc9f73e8d9